src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,27 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { Todo } from '../../props/todoProps';
+import { User } from '../../props/userProps';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+  users: User[],
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo, users }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}
```

Comment:
It is better to use libraries like classnames or clsx for conditional classnames